.eslintrc.js

```diff
@@ -37,8 +37,6 @@ module.exports = {
       require('confusing-browser-globals').filter(g => g !== 'self'),
     ),
 
-    'react/jsx-no-useless-fragment': 'error',
```

Comment:
Had to remove this rule because flags false positives (where we need fragments to make TS happy, eg wrapping an array of nodes). Overall adding fragments has no implications anyway

Reply:
Can you switch to per-line disables then? That way we catch any excess fragments in the future. (They can be annoying since they force their contents to be indented). Ideally we wouldn’t have to add the fragments too often anyway.